frontend/exporter/src/all.rs

```diff
@@ -677,13 +694,18 @@ pub struct ExpnData {
     pub collapse_debuginfo: bool,
 }
 
-#[derive(Clone, Debug, Serialize, Deserialize, JsonSchema, PartialEq)]
+#[derive(Clone, Debug, Serialize, Deserialize, JsonSchema)]
 pub struct Span {
     lo: Loc,
     hi: Loc,
     filename: FileName,
     // expn_backtrace: Vec<ExpnData>,
-    // macro_call: Option<MacCall>,
+}
+
+impl PartialEq for Span {
```

Comment:
This look equivalent to the derived implementation. Any reason you don't derive this?

Reply:
Oops, indeed, I wrote this imp for debugging purposes at some point, we should drop it (I don't have my laptop with me now though)